api/src/decorators/password-validator.decorator.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  ValidatorConstraint,
+  ValidatorConstraintInterface,
+  ValidationArguments,
+} from 'class-validator';
+import * as config from 'config';
+
+@ValidatorConstraint({ name: 'PasswordValidation', async: false })
+export class PasswordValidation implements ValidatorConstraintInterface {
+  validate(password: string, args: ValidationArguments) {
+    const minLength: number = config.get('password.minLength');
+    const includeUpperCase: boolean = config.get('password.includeUpperCase');
```

Comment:
for all values being loaded from config, handle the case when they may be a string. Use something like:

```
`${config.get('includeUpperCase')}`.toLowerCase() !== 'false'
```